src/main/java/at/hannibal2/skyhanni/utils/LorenzUtils.kt

```diff
@@ -447,4 +473,7 @@ object LorenzUtils {
         javaClass.getDeclaredField("modifiers").makeAccessible().set(this, modifiers and (Modifier.FINAL.inv()))
         return this
     }
+
+    fun Int.toBoolean() = this != 0
```

Comment:
I think we can remove those 2 methods:
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/4d0828cf-3306-4628-83cf-b40a4a3cc290)